### PR TITLE
KIC 2.0 - update railgun README.md

### PR DESCRIPTION
**Pull request by shaneutt**
The prototyping phase is over and the KIC 2.0 project is underway. This simply updates the README.md in `railgun/` to better reflect that context.

### CODE REVIEW

File: railgun/README.md [hunk 1]
```diff
@@ -1,46 +1,126 @@
-# Kubernetes Ingress Controller (KIC) Prototype
+**NOTE**: What you're seeing in this directory is the [re-architecture][kep3] of the Kong Kubernetes Ingress Controller (KIC) which will be promoted to main as part of the [KIC 2.0 Milestone][milestone-12]. Don't use this yet! If you have questions, check in with us in the [#kong channel on Kubernetes slack][slack]!
```
Comment (mflendrich) [inline railgun/README.md:1]:
```suggestion
**NOTE**: What you're seeing in this directory is the [re-architecture][kep3] of the Kong Kubernetes Ingress Controller (KIC) which will be promoted to main as part of the [KIC 2.0 Milestone][milestone-12]. `railgun/*` is not considered stable today. If you have questions, check in with us in the [#kong channel on Kubernetes slack][slack]!
```

Comment (mflendrich) [reply railgun/README.md:1]: _Don't use this under any circumstances_ comments don't age well :joy:

File: railgun/README.md [hunk 2]
```diff
@@ -1,46 +1,126 @@
...
+# using YAMLs
+$ kubectl apply -f https://bit.ly/k4k8s
+
+# or using Helm
+$ helm repo add kong https://charts.konghq.com
+$ helm repo update
+
+# Helm 3
+$ helm install kong/kong --generate-name --set ingressController.installCRDs=false
 ```
```
Comment (mflendrich) [inline railgun/README.md:65]: This doesn't install Railgun.

Comment (shaneutt) [reply railgun/README.md:65]: For now should I just remove everything except for the header and basic information?

Comment (shaneutt) [reply railgun/README.md]: switched to a minimal WIP note instead, but still removing the prototype language as we're past that stage.

**Review by mflendrich — APPROVED** [review]

**Pull request merged**